core/launch_configuration_test.go

```diff
@@ -106,7 +106,7 @@ func Test_copyBlockDeviceMappings(t *testing.T) {
 	}
 }
 
-var testSecGroupRegex = regexp.MustCompile(`^sg-[a-f0-9]{8}$`)
+var testSecGroupRegex = regexp.MustCompile(`^sg-[a-f0-9]{8,17}$`)
```

Comment:
Can't we delete this from here? It should be enough if we just use what's defined in core/autoscaling.go:850

Reply:
We need access to the same regex in the tests as well. What we could do is to define the regex as a global in autoscaling.go and then use that one in both getLaunchConfiguration() and the tests in launch_configuration_test.go. Would you prefer that?

Reply:
But why do we need it?

The tests should just hit the actual code that has the regex, unless I'm missing something.